README.md

```diff
@@ -23,32 +24,57 @@ $ go get github.com/smartcontractkit/chainlink-protos/job-distributor@v<LATEST_V
 $ go get github.com/smartcontractkit/chainlink-protos/orchestrator@v<LATEST_VERSION>
 ```
 
-## Getting Started
+### Import
+
+The import varies depending on the `go_package` option defined in the protos.
+Below is an example when `go_package` is set to `github.com/smartcontractkit/chainlink-protos/job-distributor/v1/node`
+
+```go
+import "github.com/smartcontractkit/chainlink-protos/job-distributor/v1/node"
+```
+
+## Development
+
+### Getting Started
+
+#### Requirements
```

Comment:
Should add some information here on installing task as well

Reply:
I scripted all the asdf plugin installation including task, so thats why i didn need the task section. The instruction to run the script is just below 
```
```bash
./scripts/setup-asdf-plugin.sh
asdf install
```